### ISSUE TITLE
Installer problem with config files under 'orig' symlink 

### ISSUE THREAD

**Issue by cakrit** [netdata/netdata#5039]
##### Bug report summary

I ran the netdata-installer.sh today to test a PR and saw many lines like the following:

```
File '/etc/netdata/orig/health.d/mysql.conf'  does not match stock of 'orig/health.d/mysql.conf'. Keeping it
```

The .conf files are the same, the problem is that the filename for the stock config is wrong, because it includes `orig`. The paths in `configs.signatures`

Two additional executions didn't produce the same error, so there must be a condition that is checked before comparing the md5 sums (which do match entries in `configs.signatures`. 

Since the -L option is explicitly passed to `find` at https://github.com/netdata/netdata/blob/master/netdata-installer.sh#L493 the symlink should be followed. 
e problem is around https://github.com/netdata/netdata/blob/master/netdata-installer.sh#L499-501 where `t`, the string expected in `configs.signatures` is built. 

##### Component Name
installer

##### Steps To Reproduce
Not sure how to run the installer to get it back. 

##### Expected behavior
Match the files that haven't been modified to the stock files.

**Comment by paulfantom** [comment]
As far as I know we are going to remove `configs.signatures` and validation of `/etc/netdata` files since we are storing original configuration files in `/usr/...` and not in `/etc/...`. Right now we are in a transition phase when we are waiting for users to upgrade to newer netdata so we can proceed with removal of checksum file.

**Comment by ktsaou** [comment]
I think this is the problem: https://github.com/netdata/netdata/blob/b328fde08905ad8f62cfe9381d5f1fc6d09592dd/netdata-installer.sh#L465

The installer creates that file when it migrates the conf files, to remember not to do it again. If you delete that file, migration will be re-run and you will get the warnings you get.

You get the warnings, because migration wants to match the pathname too. So, `orig/health.d/mysql.conf` does not have a stock version. `health.d/mysql.conf` does.

**Comment by paulfantom** [comment]
I don't know how to fix it. If anyone knows how to do it then create a PR.

**Comment by paulkatsoulakis** [comment]
Ok so from what i understand, i can see two quick things for this to be fixed:

1) Make sure find consistently retrieves only our conf files (Right now it brings any file type)
2) Make sure we strip the orig part too from the file, so that we can match the signatures and decide whether we keep or delete the file to overwrite.

I 'll prepare a PR so we can wrap it up there

**Comment by paulkatsoulakis** [comment]
I added the PR, but i am a bit curious as to what is the usage of the code part.
I am not sure at which point it decided to pop up when i was testing, probably on one of the runs where no stock file was removed.

https://github.com/netdata/netdata/blob/1f56d7020c44db44c9a1582f7a66f80e16d6b6af/netdata-installer.sh#L685

So i see three options to handle this:
1) Remove the creation of this link, i have a feeling it may be deprecated. We have the edit-config that does the magic for the user and we have the orig link too. I am not sure where does this extra link contribute.

2) Just ignore this specific symlink contents on the find command when we are parsing the stock configs to decide if we should remove them.

3) The installer should never end up having 0 stock files removed, so i should recheck what went wrong here and understand why this symlink popped up
